app/insights.html

```diff
@@ -0,0 +1,145 @@
+    </div>
+
+    <!-- Collect the nav links, forms, and other content for toggling -->
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <ul class="nav navbar-nav navbar-right">
+        <li>
+          <a class="page-scroll" href="journal.html">Journal</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#about">About</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#services">Services</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#privacy">Privacy</a>
+        </li>
+        <li>
+          <a class="page-scroll" href="index.html#contact">Contact</a>
+        </li>
+        <li>
+          <a class="page-scroll" id="profile_link" href="Account.html">Account</a>
+        </li>
+        <li>
+          <a class="page-scroll" id="logout_link" style="display: none" href="logout.html">Logout</a>
+        </li>
+      </ul>
+    </div>
+    <!-- /.navbar-collapse -->
+  </div>
+  <!-- /.container-fluid -->
+</nav>
+
+<div class="wrapper">
+  <div class="container">
+
+    <div class="masonry-container">
+
+      <div id="insights"></div>
+
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
+  const endpoint = 'http://localhost:3000';
+  const form = document.getElementById("new-article");
+  fetch(`${endpoint}/insights`, {
+    method: 'get',
+    headers: {
+      'Accept': 'application/vnd.api+json',
+      'Content-Type': 'application/vnd.api+json'
+    }
+  }).then(function(response) {
+    return response.json()
+  }).then(function(json) {
+    console.log('parsed json', json.data);
+    for (x in json.data) {
+      console.log(json.data[x].attributes.title);
+      document.getElementById("insights").innerHTML+=("<div class=\"card-box col-md-4 col-sm-6\">" +
+        "<div class=\"card\" data-background=\"color\" data-color=\"black\">" +
+          "<div class=\"content\">" +
+            "<h6 class=\"category\">Insight</h6>" +
+            "<h4 class=\"title\"><a href=\"#\">" + json.data[x].attributes.title + "</a></h4>" +
+            "<p class=\"description\">" + json.data[x].attributes.body + "</p>" +
+          "</div>" +
+        "</div>" +
+      "</div>")
+    }
+  }).catch(function(ex) {
+    console.log('parsing failed', ex)
+  });
+</script>
+
+<script type="text/javascript" src="https://code.jquery.com/jquery-2.2.3.min.js" integrity="sha256-a23g1Nt4dtEYOj7bR+vTu7+T8VP13humZFBJNIYoEJo=" crossorigin="anonymous"></script>
+<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/js/toastr.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/es6-shim/0.35.1/es6-shim.min.js"></script>
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/fetch/1.0.0/fetch.min.js"></script>
```

Comment:
Would the fetch polyfill and the ES6 shim be better at the top of the file? I've forgotten if these would show up as undefined in your script tag above them on older browsers or do they do weird javascript-things and hoist them so they're available?